source/tutorials/learning-paths/fundamentals/7-yaml.rst

```diff
@@ -0,0 +1,86 @@
+.. meta::
+    :description: Valohai Fundamentals learning path - Generate and update YAML configuration with valohai-utils
+
+Define your steps in Python
+###########################
+
+.. raw:: html
+
```

Comment:
```suggestion
.. admonition:: Note
    :class: seealso

    This tutorial is a part of our :ref:`learning-paths-fundamentals` series.
..
```

I'd like to have this here in the beginning of the page as well.

Reply:
On the fundamentals page there is section that gives a link to the non-utils version http://127.0.0.1:44044/tutorials/learning-paths/fundamentals/

> Valohai is completely language and framework agnostic. This quickstart is available also in Python without the valohai-utils library even though it’s strongly recommended to use it.

Good idea re: placing of the video. I'll update that